metrics/processors/node_aggregator.go

```diff
@@ -42,8 +42,7 @@ func (this *NodeAggregator) Process(batch *core.DataBatch) (*core.DataBatch, err
 				node, found := batch.MetricSets[nodeKey]
 				if !found {
 					glog.Warningf("Failed to find node: %s", nodeKey)
```

Comment:
> When can we get into this state?

If you set the metric_resolution to 10s then this error occurs. The default 30s doesn't have this issue. Not sure why exactly its happening in this situation, but the bug still remains that we trying to get the node, finding it doesn't exist, and then trying on the next line to use this nil object when we shouldn't be using it.

> Is this really a warning or lower log level?

I left the log level at its current state, if you think it should be a lower log level then we can change it.

> I'd also prefer to just continue if not found rather than if else.

We can't just continue because the next 'if' line tries to call the aggregate method with the nil node which causes a nil pointer exception. That is what this simple patch is trying to avoid.